azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -30,6 +34,16 @@
   QUEUED(110),
   FAILED_SUCCEEDED(120),
   CANCELLED(130);
+  // status is TINYINT and in H2 DB the possible values are: -128 to 127
+  // so trying to store CANCELLED in H2 fails at the moment
+
+  private static final Map<Integer, Status> numValMap = new HashMap<>();
```

Comment:
Here's the way using streams – just 2 lines: https://github.com/juhoautio/azkaban/commit/b90c4969dacfae394604ce8e6384a9fb39cfa537

Please let me know which way you prefer, maybe even some 4th way? :)